File: Resources/shuffle.py
import random 
import os


newNames = []

files = os.listdir("./test_metadata")

print(files)

old = "./test_metadata/"


counter = 1

while len(files) != 0:
    y = random.randint(0, len(files) - 1)


    os.rename(old + files[y], old + "" + str(counter) + ".json")

    newNames.append(files[y])
    counter = counter + 1
    files.pop(y)


print(newNames)

File: Resources/shuffle.js
const { readdirSync, rename } = require('fs');
const {resolve} = require('path');

const imageDirPath = resolve(__dirname, './test_metadata/');

const files = readdirSync(imageDirPath);


const old = "./test_metadata/"


let counter = 1;

while (files.length != 0){
    let y = Math.floor(Math.random() * (files.length - 1));
    console.log(y);

    rename(old + files[y], old + "" + String(counter) + ".json", (err) => {
        if (err) console.log(err)
    })

    counter = counter + 1;
    files.splice(y, 1);
}